test/support.js

```diff
@@ -15,12 +15,21 @@ var debug = require('debug')('workspace:test:support');
 
 global.Promise = require('bluebird');
 
+fileExist = function(path) {
```

Comment:
Please rename to `fileExistsSync` to prevent confusion about whether this function expects a callback or returns the result synchronously.